plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/standard/StandardDeployCommandHandler.java

```diff
@@ -122,7 +120,7 @@ private void launchDeployJob(IProject project, Credential credential)
                                               new DeployConsole.Factory());
 
     MessageConsoleStream outputStream = messageConsole.newMessageStream();
-    StandardDeployJobConfig config = getDeployJobConfig(project, credential,
+    StandardDeployJobConfig config = createDeployJobConfig(project, credential,
```

Comment:
Do you think having a separate method is clearer than inlining the builder here directly?